lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/driver/AzureBatchDriverConfiguration.java

```diff
@@ -64,8 +109,17 @@
       .bindImplementation(ResourceLaunchHandler.class, AzureBatchResourceLaunchHandler.class)
       .bindImplementation(ResourceReleaseHandler.class, AzureBatchResourceReleaseHandler.class)
       .bindImplementation(ResourceRequestHandler.class, AzureBatchResourceRequestHandler.class)
-      .bindImplementation(ResourceManagerStartHandler.class, AzureBatchRuntimeStartHandler.class)
-      .bindImplementation(ResourceManagerStopHandler.class, AzureBatchRuntimeStopHandler.class)
+      .bindImplementation(ResourceManagerStartHandler.class, AzureBatchResourceManagerStartHandler.class)
```

Comment:
What's the difference between AzureBatchRuntimeStartHandler and AzureBatchResourceManagerStartHandler?

Reply:
The base class was ResourceManagerStartHandle not RuntimeStartHandler. So I renamed it. Is there any specific reason to name it as RuntimeStartHandler?

Reply:
Got it. Just seemed strange choice of naming.